myuw/static/js/test/account_card.js

```diff
@@ -0,0 +1,26 @@
+var Global = require("./global.js");
+
+describe('AccountsCard', function(){
+    describe("Shows HR/Payroll card", function() {
+        before(function () {
+            var render_id = 'hr_payroll_card';
+
+            Global.Environment.init({
+                render_id: render_id,
+                scripts: [
+                    "myuw/static/js/card/accounts/account_card.js"
+                ],
+                templates: [
+                    'myuw/templates/handlebars/card/accounts/account_card.html'
+                ]
+            });
+
+            AccountsCard.dom_target = $('#' + render_id);
+        });
+        it("Should render instructor card", function() {
```

Comment:
what are the other test cases here?  this is one of the happy paths, but there should probably also be a test to make sure people are being rejected properly as well.  it might be easier to test those cases if the user validation code was a separate function